doc/api/http2.md

```diff
@@ -3708,6 +3713,25 @@ following additional properties:
 * `type` {string} Either `'server'` or `'client'` to identify the type of
   `Http2Session`.
 
+## Note on `:authority` and `host`
+
+HTTP/2 requires requests to have either the `:authority` pseudo-header
+or the `host` header. Using `:authority` should be preferred when
+constructing an HTTP/2 request directly, and `host` should be
+preferred when converting from HTTP/1 (in proxies, for instance).
+
+Historically, Node.js required all requests to have the `:authority`
+pseudo-header and rejected requests with the `host` header present.
+This has been corrected; the `host` header is now allowed, and the
+only requirement is for either `host` or `:authority` to be
+present (or both). The same validations from `:authority` extend
+to `host`.
```

Comment:
Not a reason to hold up landing this (as it can always be removed later), but can this paragraph be removed? Generally, docs talk about the current version.